build.gradle

```diff
@@ -28,27 +29,80 @@ nexusPublishing {
 
 group = "com.splunk"
 
+boolean snapshot = findProperty("snapshot") == 'yes'
+def otelVersion = snapshot ? "1.4.0-SNAPSHOT" : "1.3.0"
+def otelAlphaVersion = snapshot ? "1.4.0-alpha-SNAPSHOT" : "1.3.0-alpha"
+def otelInstrumentationVersion = snapshot ? "1.4.0-SNAPSHOT" : "1.3.0"
+def otelInstrumentationAlphaVersion = snapshot ? "1.4.0-alpha-SNAPSHOT" : "1.3.0-alpha"
+
+// dependencyManagement can't into qualifiers, we have to pass version the old way for deps with qualifiers
+ext.otelInstrumentationVersion = otelInstrumentationVersion
+ext.otelInstrumentationAlphaVersion = otelInstrumentationAlphaVersion
+
 subprojects {
   version = rootProject.version
 
   apply plugin: 'java'
   apply plugin: 'com.diffplug.spotless'
+  apply plugin: "io.spring.dependency-management"
   apply from: "$rootDir/gradle/spotless.gradle"
-  apply from: "$rootDir/gradle/dependencies.gradle"
 
-  repositories {
-    mavenLocal()
-    mavenCentral()
-    maven {
-      url = uri("https://oss.sonatype.org/content/repositories/snapshots")
+  dependencyManagement {
```

Comment:
In the next PR I'll move this to a convention plugin (similar to what we have in the upstream repos) and get rid of the rest of those `gradle/` scripts